### ISSUE TITLE
npm release?

### ISSUE THREAD

**Issue by limonte** [edent/SuperTinyIcons#148]
How can this awesome package be installed with npm/yarn?

Currently, I'm doing this in my `package.json`:

```json
"dependencies": {
  "super-tiny-icons": "https://github.com/edent/SuperTinyIcons"
}
```

but it would be really nice to have the proper npm release and versioning, to be able to do something like this:

```json
"dependencies": {
  "super-tiny-icons": "^1.0.0"
}
```

**Comment by limonte** [comment]
> I've never written an npm thing. Feel free to make a pull request to make it compatible.

Getting started with npm is as easy as #149 :rocket:

**Comment by edent** [comment]
Just waiting on NPM to release the name. Should be another week or two.

**Comment by edent** [comment]
It's here - but the local references have broken the README.
https://www.npmjs.com/package/super-tiny-icons
